docs/architecture/gas/parameter_definition.md

```diff
@@ -6,8 +6,68 @@ IMPORTANT: This is not the final list of parameters, it contains the base values
 which can be overwritten per protocol version. For example,
 [53.txt](core/primitives/res/runtime_configs/53.txt) changes several parameters
 starting from version 53. To see all parameter values for a specific version at
-once, check out list of JSON snapshots generated in this directory:
+once, check out the list of JSON snapshots generated in this directory:
 [core/primitives/src/runtime/snapshots](https://github.com/near/nearcore/blob/master/core/primitives/src/runtime/snapshots).
 
-<!-- TODO: How to add a new parameter -->
 <!-- TODO: Runtime Config Store -->
+
+## How to Add a New Parameter
+
+First and foremost, if you are feeling lost, open a topic in our Zulip chat
+([pagoda/contract-runtime](https://near.zulipchat.com/#narrow/stream/295306-pagoda.2Fcontract-runtime)).
+
+### Principles
+Before adding anything, please review the basic principles for gas parameters.
+- A parameter must correspond to a clearly defined workload.
+- When the workload is scalable by a factor `N` that depends on user input,
+  likely it will require a base parameter and a second parameter that is
+  multiplied by `N`. (Example: `N` = number of bytes when reading a value from
+  storage.)
+- Charge gas before executing the workload.
+- Parameters should be independent form specific implementation choices in
+  nearcore.
+- Ideally, contract developers can easily understand what the cost is simply by
+  reading the name in a gas profile.
+
+The section on [Gas Profiles](./gas_profile.md#charging-gas) explains how to
+charge gas, please also take that into considerations when defining a new
+parameter.
+
+### Necessary Code Changes
+Adding the parameter in code involves several steps.
+1. Define the parameter by adding it to the list in `parameters.txt.`
```

Comment:
```suggestion
1. Define the parameter by adding it to the list in [`parameters.txt`](https://github.com/near/nearcore/blob/master/core/primitives/res/runtime_configs/parameters.txt)
```

Reply:
I'm suggesting to either link it to the repo, or to do as in (2), where the path is also specified.

Reply:
This file is already linked once on this page. So I added just the path.